src/network/networkctl.c

```diff
@@ -1130,6 +1180,91 @@ static int link_status_one(
                         return r;
         }
 
+        if (streq_ptr(info->netdev_kind, "bridge")) {
+                if (info->forward_delay) {
+                        r = table_add_cell(table, NULL, TABLE_EMPTY, NULL);
+                        if (r < 0)
+                                return r;
+                        r = table_add_cell(table, NULL, TABLE_STRING, "forward_delay:");
```

Comment:
Please capitalize the field name: `Forward Delay:`, `Hello Time:`, or `Max Age:`.